### ISSUE TITLE
Script doesn't seem to be running

### ISSUE THREAD

**Issue by raxod502** [mgsloan/squarespace-escape-suppressor#1]
Hello! I've installed the extension from https://chrome.google.com/webstore/detail/squarespace-escape-suppre/ecbamndhnebdcfmidendannaficioean?hl=en, as it's just what I was looking for. So far, however, it doesn't seem to be doing anything.

<details>
<summary>Screenshot of extension page in settings</summary>

![image](https://user-images.githubusercontent.com/6559064/99815205-dd6e8680-2afe-11eb-9280-bfe6fb84f2b2.png)

</details>

In the console I don't see anything logged by the extension.

<details>
<summary>Screenshot of console</summary>

![image](https://user-images.githubusercontent.com/6559064/99815362-0727ad80-2aff-11eb-897a-5573bbcedf07.png)

</details>

The Squarespace website in question is https://kelifern-pomeranz.squarespace.com/. From the Inspect Element view, I'm not seeing any hits when I search for `data-controllers-bound` on this page; is that the issue?

OS: Kubuntu 20.04.1 LTS
Browser: Google Chrome 86.0.4240.75-1

**Comment by mgsloan** [comment]
Thanks for reporting!  I've changed the site check to something that should be more reliable, included in version 3.  Now it looks for a ` This is Squarespace. ` comment in the first few elements of the `<head>` node.

Also, I recognize your name!  I made this change in emacs with packages managed by [straight.el](https://github.com/raxod502/straight.el) - thanks a bunch for making that!

**Comment by raxod502** [comment]
Nice! I can confirm the fix.

> I made this change in emacs with packages managed by straight.el - thanks a bunch for making that!

Wow, what a small world! You're welcome and I'm glad you've enjoyed it. By the way, I am also from Boulder! I didn't recognize any specific locations but the scenery in https://mgsloan.com/posts/tree-based-computing/ definitely looks familiar:

<p align="center"><img src="https://user-images.githubusercontent.com/6559064/99976410-44d34300-2d58-11eb-84cd-ed907f7c226a.png" width="500px"></p>

I was also amused to see the system administration log https://github.com/mgsloan/mgsloan-dotfiles/blob/master/env/setup-log.md as I keep one too (although somewhat more verbose at around 15,000 lines of Markdown, since I don't have as much faith in my ability to recreate system configurations).

**Comment by mgsloan** [comment]
Cool!  Small world indeed.  Boulder has been great, I've lived around here
for about a year and a half now.

Yes that photo is by the dakota ridge / mt sanitas trails.  Behind some
rock, so not an obtrusive spot to compute, I think barely anyone noticed.
My old house was a few minutes away from one of the mt sanitas trailheads!

Yeah the setup log is super helpful for knowing what mutations I've made to
the machine, and for setting up a new machine.  Maybe I should learn nix,
but so far this approach has satisficed.
